hooks/tk-houdini_actions.py

```diff
@@ -116,4 +137,147 @@ def _merge(self, path, sg_publish_data):
         hou.hipFile.merge(path,
                           node_pattern="*",
                           overwrite_on_conflict=False,
-                          ignore_load_warnings=False)       
+                          ignore_load_warnings=False) 
+
+
+    ##############################################################################################################
+    def _import(self, path, sg_publish_data):
+        """Import the supplied path as a geo/alembic sop.
+        
+        :param str path: The path to the file to import.
+        :param dict sg_publish_data: The publish data for the supplied path.
+        
+        """
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembic")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = _get_current_context("/obj")
+        app.log_debug("Creating geo: %s" % (name,))
```

Comment:
Would be good to mention the path to the context it'll be created in.  Just for debugging help.